packages/vitest/src/node/cli-api.ts

```diff
@@ -56,6 +56,9 @@ export async function startVitest(
   if (typeof options.browser === 'object' && !('enabled' in options.browser))
     options.browser.enabled = true
 
+  if ('threads' in options && options.experimentalVmThreads)
```

Comment:
`'threads' in options` checks if the key exists right? How about the value?

Reply:
Value doesn't matter, I mention in docs that both threads and no-threads will throw an error